test/crsstrings.jl

```diff
@@ -0,0 +1,35 @@
+@testset "PROJJSON" begin
+  jsonstr = GeoIO.projjsonstring(EPSG{4326})
+  @test isvalidprojjson(JSON3.read(jsonstr))
```

Comment:
Isn't this test already covered in the list below?

Reply:
Yes I just thought all that testing construction is complicated and it might not be obvious how to simply run one test.

Reply:
We can remove this one then. The loop with all codes is clear enough 👍🏽